test/e2e/common/util.go

```diff
@@ -45,6 +45,13 @@ const (
 	NodeE2E Suite = "node e2e"
 )
 
+var (
+	// non-Administrator Windows user used in tests. This is the Windows equivalent of the Linux non-root UID usage.
+	nonAdminTestUserName = "ContainerUser"
+	// non-root UID used in tests.
+	nonRootTestUserID = int64(1000)
```

Comment:
now we can only set userId as 1000 for linux. For current tests, it is fine. Will it become a concern in the future?

Reply:
Nothing changed, userid 1000 was also used before this change. Other userids can still be used.

Reply:
Here it is kind of hardcoded to set user id to 1000. My point is all tests will have to share the exact same value which is defined here.

Reply:
I don't see the problem.

Reply:
@jingxu97 -  Before this value was split across many places, and this change defines it once. Are you saying you'd prefer the old way where 1000 was individually hardcoded in many test cases?